adapter-s4j/src/main/resources/scenarios/nbs4j_msg_proc_named.yaml

```diff
@@ -0,0 +1,28 @@
+scenarios:
+  msg_send:
+    - run driver=s4j cycles=100 threads=4 num_conn=2 num_session=2 session_mode=client_ack config=../conf/s4j_config.properties service_url=pulsar://localhost:6650 web_url=http://localhost:8080
```

Comment:
It would be preferable to use named steps. To do this, simply replace the list with a map, where the keys are the step names.

Reply:
Thanks. Changed to named steps.